ChatQnA/docker_compose/amd/gpu/rocm/compose.yaml

```diff
@@ -0,0 +1,182 @@
+# Copyright (C) 2024 Advanced Micro Devices, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+services:
+  chatqna-redis-vector-db:
+    image: redis/redis-stack:7.2.0-v9
+    container_name: redis-vector-db
+    ports:
+      - "${CHATQNA_REDIS_VECTOR_PORT}:6379"
+      - "${CHATQNA_REDIS_VECTOR_INSIGHT_PORT}:8001"
+  chatqna-dataprep-redis-service:
+    image: ${REGISTRY:-opea}/dataprep-redis:latest
```

Comment:
${REGISTRY:-opea}/llm-docsum-tgi:${TAG:-latest}

Reply:
Done, tag reverted to ${TAG:-latest}